app/models/comment.rb

```diff
@@ -2,7 +2,7 @@ class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: :author_id
   belongs_to :post
 
-  after_save :comment_counter_update
+  after_create :comment_counter_update
 
   def comment_counter_update
```

Comment:
- [ ] The changes made in the comment spec would fail still. To fix that you still need to modify this method. I would recommend using this instead:
 
  ```
  def comment_counter_update
    post.increment!(:comments_counter)
  end
  ```